Brain/src/model/train_model.py

```diff
@@ -0,0 +1,24 @@
+"""train model:
+{
+    "id": "String",
+    "data": [{"page_content": "String", "timestamp": 0}],
+    "status": "created | updated | deleted",
+}"""
+
+from src.model.requests.request_model import Train
+
+
+class TrainModel:
+    def __init__(self, TrainData: Train):
```

Comment:
the variable name syntax should be lowercase with underline(_) to make it more semantic with multi-nouns
Ex: TrainData -> train_data

Reply:
done.